physical/raft/raft.go

```diff
@@ -127,13 +127,6 @@ type RaftBackend struct {
 	// However, only active node will have a "running" autopilot.
 	autopilot *autopilot.Autopilot
 
-	// disableAutopilot if set will not put autopilot implementation to use. The
-	// fallback will be to interact with the raft instance directly. This is useful
-	// to retain state of operations on existing clusters which do not have
-	// autopilot enabled yet. For new clusters however, the default will be to have
-	// autopilot enabled from get-go.
-	disableAutopilot bool
```

Comment:
Is there still a way to disable?

Reply:
There is now. Using `VAULT_RAFT_AUTOPILOT_DISABLE` env var. It requires a restart to take effect though.